codespell_lib/_codespell.py

```diff
@@ -1119,7 +1125,9 @@ def main(*args: str) -> int:
 
     exclude_lines: Set[str] = set()
     if options.exclude_file:
-        build_exclude_hashes(options.exclude_file, exclude_lines)
+        for exclude_file_list in options.exclude_file:
+            for exclude_file in exclude_file_list.split(","):
+                build_exclude_hashes(exclude_file, exclude_lines)
```

Comment:
Remove the `if options.exclude_file:` line above and then
```suggestion
    for exclude_file_list in options.exclude_file or []:
        for exclude_file in exclude_file_list.split(","):
            build_exclude_hashes(exclude_file, exclude_lines)
```